src/anyon/qpu_interface.jl

```diff
@@ -254,16 +305,29 @@ get_requestor(client::Client) = client.requestor
 
 
 """
-    submit_job(client::Client, circuit::QuantumCircuit, shot_count::Integer, project_id::String, machine_name::String)
+    submit_job(
+        client::Client,
+        circuit::QuantumCircuit,
+        shot_count::Integer,
+        project_id::String,
+        machine_name::String
+    )
 
-Submit a circuit to a `Client` of `QPU` service, requesting a 
-particular machine (`machine_name`), a number of repetitions (`shot_count`). 
-Returns circuitID.
+Submit a circuit to a `Client` that a connects to a `QPU` service for a particular machine
```

Comment:
Submit a circuit using a `Client`, which connects to a ...

Reply:
I'm assuming that what's identified by `machine_name` is the `QPU` service and not the host server.

Reply:
oh, yes that is right